src/Dropdown/Dropdown.js

```diff
@@ -358,7 +367,7 @@ Dropdown.Footer = createComponent({
     as: 'footer',
   }),
   style: ({ theme }) => css`
-    background: ${theme.colors.grayLightest};
+    background: ${({theme}) => get(theme, 'colors.grayLightest', 'gray')};
```

Comment:
These changes are hacks to enable enzyme testing.

Reply:
do we have to hack this for all components we want to test?

Reply:
Perhaps, it's worth looking into moving forward if there's a more elegant way to handle this, assuming we turn top enzyme for more testing.